docs/exchange-operators/rosetta/samples/scan-blocks.mdx

```diff
@@ -21,8 +43,29 @@ def wait_for_block(block_index):
         latest_block = network_status()["current_block_identifier"]["index"]
 ```
 
+</TabItem>
+</Tabs>
+
 It can be used to scan blocks like this:
 
+<Tabs>
+<TabItem value="ts" label="TypeScript">
+
+```ts
+let latestBlockHeight = (await network_status()).current_block_identifier.index
+while (true) {
+    const lastBlock = waitForBlock(latestBlockHeight)
+    
+    # some processing according to business logic
```

Comment:
this is python comment in ts code

Reply:
I'd suggest addressing these comments as a separate PR once main has been merged